src/bioregistry/data/bioregistry.json

```diff
@@ -32837,6 +32849,7 @@
       "wikidata": "P4229"
     },
     "name": "International Classification of Diseases, 10th Revision, Clinical Modification",
+    "pattern": "^[A-TV-Z][0-9][0-9AB]\\.?[0-9A-TV-Z]{0,4}$",
```

Comment:
Might need to refine this. See https://medium.com/@manabu.torii/regex-pattern-for-icd-10-cm-codes-5763bd66e26d